README.md

```diff
@@ -1,221 +1,274 @@
+* [Getting Started](#getting-started)
+* [Demo](#demo)
+* [Components](#components)
+* [What To Do Next?](#what-to-do-next)
+* [Code of Conduct](#code-of-conduct)
 
-This controller watches for relevant changes to pods, identities, and bindings through the API server. When a change is detected it runs a loop to check if there are actions to be performed. Based on the change detected MIC would either add or delete assigned identities. In case of user assigned identity usage, MIC is responsible for assigning it to the underlying VM in which the pod gets scheduled during pod creation. During pod deletion it would remove those identities from the VM. Similar steps are taken by MIC when identities or bindings are created or deleted.
+## Getting Started
 
-### Node Managed Identity(NMI)
+### Prerequisites
 
-The authorization request for fetching Service Principal Token from MSI endpoint is sent to a standard Instance Metadata endpoint which is redirected to the NMI pod by adding ruled to redirect POD CIDR traffic with metadata endpoint IP on port 80 to be sent to the NMI endpoint. The NMI server identifies the pod based on the remote address of the request and then queries the k8s (through MIC) for a matching Azure identity. It then makes an Azure Active Directory Authentication Library ([ADAL](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-authentication-libraries)) request to get the token for the client id and returns as a response to the request. If the request had client id as part of the query it validates it against the admin configured client id.
+You will need a Kubernetes cluster running on Azure, either managed by [AKS] or provisioned with [AKS Engine].
 
-Similarly a host can make an authorization request to fetch Service Principal Token for a resource directly from the NMI host endpoint (http://127.0.0.1:2579/host/token/). The request must include the pod namespace `podns` and the pod name `podname` in the request header and the resource endpoint of the resource requesting the token. The NMI server identifies the pod based on the `podns` and `podname` in the request header and then queries k8s (through MIC) for a matching azure identity.  Then NMI makes an ADAL request to get a token for the resource in the request, returns the `token` and the `clientid` as a response to the request.
+### 1. Create the Deployment
 
-An example cURL command:
+AAD Pod Identity consists of the Managed Identity Controller (MIC) and Node Managed Identity (NMI) components in a Kubernetes deployment. For more information, see [Components].
 
-```bash
-curl http://127.0.0.1:2579/host/token/?resource=https://vault.azure.net -H "podname: nginx-flex-kv-int" -H "podns: default"
-```
+Run this command to create the `aad-pod-identity` deployment on an RBAC-enabled cluster:
 
-## Getting Started
+```shell
+kubectl create -f https://raw.githubusercontent.com/Azure/aad-pod-identity/master/deploy/infra/deployment-rbac.yaml
```

Comment:
Should we use `apply` instead?

Reply:
I can't decide... `kubectl apply` is more flexible and probably preferable in general, but this is a "Getting Started" section where arguably `kubectl create` is preferable because we are explicitly creating new resources and might want an error rather than overwriting them.

It's easy to change this if you have a strong preference, although I think we would also want to change the preceding wording to "create or update" in those cases.

Reply:
I changed it to `apply` after more consideration.